st2api/st2api/controllers/v1/rules.py

```diff
@@ -70,12 +70,14 @@ class RuleController(resource.ContentPackResourceController):
     @request_user_has_permission(permission_type=PermissionType.RULE_LIST)
     @jsexpose()
     def get_all(self, **kwargs):
-        return super(RuleController, self)._get_all(**kwargs)
+        from_model_kwargs = {'ignore_missing_trigger': True}
+        return super(RuleController, self)._get_all(from_model_kwargs=from_model_kwargs, **kwargs)
 
     @request_user_has_resource_db_permission(permission_type=PermissionType.RULE_VIEW)
     @jsexpose(arg_types=[str])
     def get_one(self, ref_or_id):
-        return super(RuleController, self)._get_one(ref_or_id)
+        from_model_kwargs = {'ignore_missing_trigger': True}
```

Comment:
It's the "generic" argument to `from_model` which also applies to get one, not just list...